lib/qrda-import/patient_importer.rb

```diff
@@ -110,11 +110,12 @@ def normalize_references(patient, entry_id_map)
 
           relations_to_add = []
           data_element.relatedTo.each do |related_to|
-            relations_to_add << entry_id_map["#{related_to.value}_#{related_to.namingSystem}"]
+            relations_to_add += entry_id_map["#{related_to.value}_#{related_to.namingSystem}"]
           end
-          data_element.relatedTo.destroy
+          # Clear out relatedTo element. We will be replacing the ids from the QRDA document with our interal ids.
+          data_element.relatedTo = []
```

Comment:
looks like these 5 lines of code could be simplified to `data_element.relatedTo = relations_to_add.map{ |r| r.to_s }` ?